AlpsSDK/AlpsManager/AlpsManager.swift

```diff
@@ -20,23 +28,22 @@ open class AlpsManager: MatchMonitorDelegate, ContextManagerDelegate {
         }
     }
     
-    lazy var contextManager = ContextManager(delegate: self)
-    lazy var matchMonitor = MatchMonitor(delegate: self)
-    var onMatch: ((_ matches: [Match], _ device: Device) -> Void)?
+    public lazy var contextManager = ContextManager(delegate: self)
+    public lazy var matchMonitor = MatchMonitor(delegate: self)
     
-    lazy var mobileDevices = MobileDeviceRepository()
-    lazy var pinDevices = PinDeviceRepository()
+    public lazy var mobileDevices = MobileDeviceRepository()
+    public lazy var pinDevices = PinDeviceRepository()
     
-    lazy var publications = PublicationRepository()
-    lazy var subscriptions = SubscriptionRepository()
+    public lazy var publications = PublicationRepository()
+    public lazy var subscriptions = SubscriptionRepository()
     
-    lazy var locationUpdateManager = LocationUpdateManager()
+    public lazy var locationUpdateManager = LocationUpdateManager()
 
-    init(apiKey: String, baseUrl: String? = nil) {
+    public init(apiKey: String, baseURL: String? = nil) {
```

Comment:
I also made public almost all the members and many class, when using SDK to try Proximity Handler.
We can keep them all open for the moment and we will look for private when we talk about security

Reply:
Agreed.